blog/2024/09/juliacon-2025-announcement.md

```diff
@@ -0,0 +1,98 @@
+# JuliaCon Global 2025 and JuliaCon 2024 Wrap-Up
+
+![Pittsburgh Skyline](/assets/blog/2024-JuliaCon/AdobeStock_50245829.jpeg)
+We’re happy to announce that JuliaCon Global 2025 will be held in Pittsburgh Pennsylvania, USA in the ==third week of July==! As we approach the conference next year we will make further announcements.
+
+A few important date estimates for you:
+
+- Call for Proposals will be open November 1 - January 31
+- Volunteer, and scholarship applications will be open from December 1 - February 28
+- Early-bird tickets will go on sale late this year, full-price ticket sales will start on ==March 15==
+
+We look forward to seeing you in Pittsburgh, and please keep an eye out for future announcements!
+
+***If you’re interested in joining the JuliaCon Global 2025 organizing committee please fill out this form https://forms.gle/1V5nkbeGr959R8s66***
+
+## Julia Community Efforts
+
+We have a couple new community efforts we’d like to talk about!
+
+### Julia Online Videos
+
+We have a lot of community members who want to give presentations at JuliaCon each year but can’t make it. We receive a lot of requests to submit videos to the conference, but for now we intend to run JuliaCon as an in-person conference first and foremost. Hybrid conferences take a lot of effort, and have a negative impact on the in-person experience.
+
+As an alternative we are starting a new trial program for community members to propose videos for upload to the Julia YouTube Channel. If your proposal is accepted you’ll be asked to prepare your proposal into a full video for upload! This call for proposals will be evergreen, it will be open for as long as we run this program so you are welcome to propose a video at any time.
+
+Our YouTube channel is a fantastic forum for you to advertise your packages / work, present tutorials which drive users to the Julia ecosystem, or to present unique perspectives on your favorite problem in computer science, software development, or scientific computing! We have well over 100k views per month, and a lot of impressions on each video we upload.
+
+If this program interests you please fill out the following form:
+
+https://forms.gle/vHUsG4hbbVf5V472A
+
+### JuliaCon Locals
```

Comment:
Or "JuliaCons Local"? 🤔

Reply:
Eh, as much as Attorneys General is fun I think it's kinda hard to parse. And the event as a whole is called JuliaCon Local

Is it more correct?